pkg/api/types.go

```diff
@@ -1566,6 +1566,11 @@ type PodSpec struct {
 	// If specified, the fully qualified Pod hostname will be "<hostname>.<subdomain>.<pod namespace>.svc.<cluster domain>".
 	// If not specified, the pod will not have a domainname at all.
 	Subdomain string `json:"subdomain,omitempty"`
+	//Optional: Docker "--shm-size" support. Defines the size of /dev/shm in the IPC namespace of the pod.
```

Comment:
we are generally trying to avoid docker specific defaulting behaviors in the API as we move towards the CRI model supporting multiple container runtimes.  docker appears to default to 64MB, so if we did the same, the defautling would have to happen in our own defaults.go workflow.